OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,58 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+struct dynamically_sized_perf_event_call_chain_sample {
+  struct dynamically_sized_perf_event_call_chain {
+    uint64_t dyn_size;
+    std::unique_ptr<uint64_t[]> data;
```

Comment:
If this is called "data" and can hold (more or less) generic data, I'd expect the type here to be char and not uint64_t.

Reply:
Or, if it is not generic data but actually the pointers in the call chain, then change the name to indicate that. I'd also make it clear and rename dyn_size to callchain_length or something like that.

Reply:
It is holding the instruction pointers, that are in fact 64 bit of size. I'll rename the field here.

Reply:
The idea of these "dynamically sized" structs is to mimic the layout of `perf_event_open` records, hence you should keep the types and the names used in the manpage. Otherwise, have the appropriate fields directly in the `SampleFPPerfEvent`.

Reply:
In the manpage its called `nr` , and `ips`. I'll name it that way.